### PR TITLE
Added cohort big red button

### PR DESCRIPTION
**Pull request by dan1212111**
Screenshot 2022-05-09 at 11 42 12

- Header `add cohort Button` directs to the **add cohort page**
- Button **creates cohort** and then directs the user to **viewCohort** page

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -1,84 +1,84 @@
...
-    localStorage.setItem(process.env.REACT_APP_USER_TOKEN, '');
-    navigate('../login', { replace: true });
-  };
-
-  const addCohortHandle = () => {
-    client.post('/cohort', {}).catch((err) => console.log(err.response));
-  };
+    event.preventDefault()
+    localStorage.setItem(process.env.REACT_APP_USER_TOKEN, "")
+    navigate("../login", { replace: true })
```
Comment (Auenc) [inline src/components/Header/Header.jsx:17]: These strings should use single quotes.

File: src/App.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ import PostsPage from "./components/posts/PostsPage";
 import EditUser from "./components/users/userProfile/EditUserProfile";
 import ProfilePage from "./components/users/userProfile/UserProfile";
 import ViewCohort from "./components/users/viewCohorts/viewCohorts"
```
[the remark this answers is not in the document]
Comment (dan1212111) [reply src/App.js:8]: Sorry my it's my formatter 😬

File: src/components/Header/Header.jsx [hunk 3]
```diff
@@ -1,84 +1,84 @@
...
+  }
 
   const handleMyProfileLink = () => {
-    const userId = localStorage.getItem('userId');
-    navigate(`../user/${userId}`);
-  };
+    const userId = localStorage.getItem("userId")
```
Comment (Auenc) [inline src/components/Header/Header.jsx:21]: Can we change the strings in this file to use single quotes and have semi-colons at the end of each line, please?

**Review by Auenc — CHANGES_REQUESTED** [review]
The code looks good, but you're using the wrong quote style, and don't have semi-colons and the end of every line of code. 

Please change the quote style and add the semi-colons before we can merge.

**Review by Auenc — DISMISSED** [review]

**Review by marianphillips — APPROVED** [review]

**Review by KinTale — APPROVED** [review]

**Review by marianphillips — APPROVED** [review]

**Pull request merged**